eng/versioning/version_client.txt

```diff
@@ -5,12 +5,12 @@ com.azure:azure-sdk-all;1.0.0;1.0.0
 com.azure:azure-sdk-parent;1.6.0;1.6.0
 com.azure:azure-client-sdk-parent;1.7.0;1.7.0
 com.azure:azure-ai-textanalytics;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-core;1.1.0;1.2.0-beta.1
+com.azure:azure-core;1.1.0;1.2.0
 com.azure:azure-core-amqp;1.0.0-beta.9;1.0.0-beta.9
-com.azure:azure-core-http-netty;1.1.0;1.2.0-beta.1
-com.azure:azure-core-http-okhttp;1.1.0-beta.1;1.1.0-beta.1
+com.azure:azure-core-http-netty;1.1.0;1.2.0
+com.azure:azure-core-http-okhttp;1.1.0-beta.1;1.1.0
 com.azure:azure-core-management;1.0.0-beta.8;1.0.0-beta.8
-com.azure:azure-core-test;1.1.0-beta.1;1.1.0-beta.2
+com.azure:azure-core-test;1.1.0-beta.2;1.1.0
```

Comment:
This is fine to use a beta version as the dependency version since it is a test only dependency, in the future we should make sure this locks into a GA'd version.